app/Mail/VoucherDuplicateEnteredEmail.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class VoucherDuplicateEnteredEmail extends Mailable
+{
+    use Queueable, SerializesModels;
+
+    public $user;
+    public $trader;
+    public $voucher;
+    public $vouchercode;
+    public $market;
+
+    /**
+     * Create a new message instance.
+     */
+
+    public function __construct($user, $trader, $voucher)
+    {
+        $this->user = $user->name;
+        $this->trader = $trader->name;
+        $this->voucher = $voucher;
+        $this->market = $trader->market;
```

Comment:
@katjam @charlesstrange2 I've updated and tests pass locally but Travis is shouting if there's an apostrophe in the copy.